### ISSUE TITLE
Give user a way to mute or "ACK" specific issue/path/type triplets in the activity list

### ISSUE THREAD

**Issue by posita** [owncloud/client#10296]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Feature description

Having the blue "I" leads to confusion for users (see #8970) and reduces otherwise valuable signal. I have paths in my in ownCloud directory that won't be synced for various reasons (e.g., they are symbolic links, or there might be a `node_modules` directory that I often trigger the temporary creation of, etc.). Further, I am totally cool with those things not being synced, but don't want them to exist anywhere else (i.e., outside my ownCloud directory). The presence of those files currently guarantees that spotting **_new_** issues that don't involve those known files will be hard, since the blue "I" is perpetually displayed.

I would like the ability to whitelist issue/path/type (folder, file, symlink) triplets so that they are ignored for the purpose of activity issue reporting, including whether the blue "I" appears. This is important, since it would allow me to achieve a green check for issues I already know about and accept, while still alerting me to knew issues I may need to investigate when the blue "I" reappears.

### Proposed solution

I would like for the ability to right click on an issue that appears in the `Not Synced` activity log and choose something like "ignore this issue for the [folder/file/symlink] at this path". I would like the desktop to keep track of these preferences as a purely local setting (i.e., do not affect behavior of any other client, do not store the settings on the server, etc.). From that point on, that desktop client instance should ignore that particular issue for that path and type. If a path involved in a white-listed issue/path/type triplet changes type or is deleted, the triplet should automatically be removed from the white-list.

There should also probably be an interface where all currently white-listed issue/path/type triplets can be seen, and where any existing entry can be removed from the white-list manually.

### Alternative solutions you considered

This is probably related to #8970, but that issue is closed for additional comment and I'm not sure https://github.com/owncloud/client/issues/8970#issuecomment-1293110225 adequately captures the feature I'm talking about. Current file pattern ignore lists do not tackle this issue. They only ignore files, but those ignored files turn up as issues in the activity list.

**Comment by posita** [comment]
@jnweiger, @aNDy-Squirrel, @avrhack, and @The9Cat, can you give a thumbs up here if the above feature (if implemented) would it address the concerns you each raise in #8970? If so, just a thumbs up will do. If not, maybe a brief explanation of any concern I missed would be helpful. I'd like to keep this concise and focused on a solution, if at all possible.

**Comment by posita** [comment]
@MichaKersloot and @SamuAlfageme, feel free to chime in here as well. (Apologies for missing you in my original call-out. I did not intend to exclude anyone.)

**Comment by MichaKersloot** [comment]
It would require some knowledge from the end user about what the issue is and requires an action he/she probably doesn't really understand. But having a method to solve the blue exclamation, is way better than the current situation!

**Comment by posita** [comment]
What does "stale" mean?

**Comment by posita** [comment]
Nope. **_Still_** not stale. **_Still_** an issue. Please remove the label.

**Comment by posita** [comment]
STILL NOT STALE.
